### PR TITLE
Add gradle maven

### PR DESCRIPTION
**Pull request by lesv**
I'll need to update the version here and everywhere soon.  This is just getting this checked in.

### CODE REVIEW

File: helloworld-compat/src/main/java/com/example/appengine/helloworld/HelloServlet.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import javax.servlet.http.HttpServletResponse;
+
+// [START example]
+@SuppressWarnings("serial")
+public class HelloServlet extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest req, HttpServletResponse resp) throws IOException {
+    PrintWriter out = resp.getWriter();
+    out.println("Hello, world - Flex Compat");
```
Comment (tswast) [inline helloworld-compat/src/main/java/com/example/appengine/helloworld/HelloServlet.java]: Should this do something that uses the App Engine SDK, like https://github.com/GoogleCloudPlatform/appengine-java-vm-hello/blob/master/src/main/java/com/google/appengine/demos/hello/server/HelloInfo.java ?

File: travis.sh [hunk 2]
```diff
@@ -35,7 +35,8 @@ else
   echo 'External PR: skipping integration tests.'
 fi
 ./java-repo-tools/scripts/test-localhost.sh jetty helloworld-jsp
-./java-repo-tools/scripts/test-localhost.sh jetty helloworld-servlet
+# ./java-repo-tools/scripts/test-localhost.sh jetty helloworld-servlet
```
Comment (tswast) [inline travis.sh]: I guess we will need to update the script to work with the new plugins. Is that one of the changes you were talking about?

File: helloworld-compat/src/main/java/com/example/appengine/helloworld/HelloServlet.java (hunk 1 above)
Comment (lesv) [inline helloworld-compat/src/main/java/com/example/appengine/helloworld/HelloServlet.java]: Yes - a later improvement, but not for today.

**Comment by tswast** [comment]
LGTM

File: helloworld-compat/src/main/java/com/example/appengine/helloworld/HelloServlet.java (hunk 1 above)
Comment (lesv) [inline helloworld-compat/src/main/java/com/example/appengine/helloworld/HelloServlet.java]: done.

File: travis.sh (hunk 2 above)
Comment (lesv) [inline travis.sh]: Yep

Comment (lesv) [inline travis.sh]: definitely a TBD

**Pull request merged**